tests/unit/executors/test_set_metas.py

```diff
@@ -48,3 +49,30 @@ def test_default_args_from_python():
     assert isinstance(b.metas, SimpleNamespace)
     # name is always auto-assigned
     assert b.metas.name
+
+
+def test_name_python_jaml_identical():
+    # There are two different ways of importing the executors in jina 2.0.
+    # We want the executors to have the same metas.name field regardless of
+    # the way they were imported!
+
+    # First way of import using py_modules argument in jaml file
+    from jina.jaml.helper import load_py_modules
+    from jina.helper import typename
+
+    load_py_modules({'py_modules': ['metas_executors.py']})
+    from metas_executors import TestExecutor
+
+    jaml_executor = TestExecutor()
+    jaml_metas_name = jaml_executor.metas.name
+    jaml_typename = typename(jaml_executor)
+
+    # Second way of importing directly via path in python
+    from .metas_executors import TestExecutor
+
+    py_executor = TestExecutor()
+    py_metas_name = py_executor.metas.name
+    py_typename = typename(py_executor)
+
+    assert jaml_metas_name == py_metas_name
```

Comment:
and what is the expected value for each of these?

Reply:
https://github.com/jina-ai/jina/pull/2486/commits/e42c536831e118a7e5232501ce87de24b2472fae